charts/rancher/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
 name: rancher
 description: Install Rancher Server to manage Kubernetes clusters across providers.
-version: 2.7.6
-appVersion: v2.7.6
-kubeVersion: < 1.27.0-0
+version: 2.7.7
+appVersion: v2.7.7
+kubeVersion: < 1.28.0-0
```

Comment:
this freaks me out so much. patch version of rancher supported a new minor (kube will be forever 1.x) version of k8s.